client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -16,16 +21,28 @@ const PlansIntervalToggleLabel = styled.span`
 	margin-right: 10px;
 `;
 
+const PluginAnnualSavingLabelMobile = styled.span< PluginAnnualSavingLabelProps >`
+	color: ${ ( props ) =>
+		props.isSelected ? 'var( --studio-white-100 )' : 'var( --studio-green-60 )' };
+`;
+
+const PluginAnnualSavingLabelDesktop = styled.span`
+	font-size: 12px;
+	color: var( --studio-green-60 );
+`;
+
 type Props = {
 	onChange: ( selectedValue: 'MONTHLY' | 'ANNUALLY' ) => void;
 	billingPeriod: IntervalLength;
 	compact: boolean;
+	plugin?: object;
```

Comment:
Using `object` as a type is not ideal, since it doesn't tell us anything about the data. If you don't know what the data is, you can use `unknown`. However, in this case I think we do know what it is, since I see it being used below like `plugin?.variations?.yearly?.product_slug`.

```suggestion
	plugin?: { variations?: { yearly?: { product_slug: string }; monthly?: { product_slug: string } } };
```